core/controllers/contributor_dashboard_test.py

```diff
@@ -489,12 +489,12 @@ def test_get_reviewable_translation_opportunities_when_state_is_removed(
         self
     ):
         # Create a new exploration and linked story.
-        multiple_choice_state_name = 'Multiple choice state'
+        continue_state_name = 'continue state'
```

Comment:
This change is fine, but I'm just curious ... why was it needed?

Reply:
Yeah, so when the multiple-choice interaction gets initialized it has empty values, and internally when the validation function gets called, it raises an error that the `choices should not be empty` So, just to avoid this completely I changed the interaction.
Thanks